src/importexport/musicxml/tests/data/testBreaksImplExpl_all_ref.xml

```diff
@@ -134,7 +134,7 @@
       <print new-page="yes"/>
       <direction>
         <direction-type>
-          <words font-family="Edwin">After system break</words>
```

Comment:
These had been added not long ago and are not in the MU3 reference set.
However I am slightly concerned there's actually no mention at all of "Edwin" in most of the xml files, whereas the MU3 reference set did have "words-font" and "lyrics-font" entries specifying the defaults. Will investigate.